.chglog/config.yml

```
@@ -0,0 +1,28 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/go-vela/vela-terraform
```

I think we'll want to update this to point at this repo 👍 

```suggestion
  repository_url: https://github.com/go-vela/secret-vault
```